tsconfig.json

```diff
@@ -17,6 +17,6 @@
     // Advanced
     "skipLibCheck": true // Don't type check .d.ts files, which mostly come from third party libraries.
   },
-  "include": ["**/*", ".storybook/*", ".eslintrc.js"],
+  "include": ["**/*", ".storybook/*", ".eslintrc.js", "src/components/grid-aware/Footer/SubscriptionBlock/.js"],
```

Comment:
What is this change for? `**/*` should already include basically everything, and because `src/components/grid-aware/Footer/SubscriptionBlock/.js` is just a simple string, not a glob pattern, this won't match any actual files.